app/(auth)/lib/get-current-subscription.ts

```diff
@@ -1,29 +1,22 @@
 "use server";
 
-import {
-	db,
-	subscriptions,
-	supabaseUserMappings,
-	teamMemberships,
-	teams,
-} from "@/drizzle";
+import { db, supabaseUserMappings, teamMemberships, teams } from "@/drizzle";
 import { getUser } from "@/lib/supabase";
 import { eq } from "drizzle-orm";
 
-export const getUserSubscriptionId = async () => {
+export const getUserTeamId = async () => {
 	const user = await getUser();
 	// TODO: When team plans are released, a user may belong to multiple teams, so we need to handle that case.
 	// e.g., fetch team id through agents or so.
-	const [subscription] = await db
-		.select({ id: subscriptions.dbId })
-		.from(subscriptions)
-		.innerJoin(teams, eq(teams.dbId, subscriptions.teamDbId))
+	const [team] = await db
+		.select({ teamId: teams.dbId })
```

Comment:
📝 My Notes:

I have confirmed the association between `teams.dbId` and `subscriptions` in the following sections.

https://github.com/giselles-ai/giselle/blob/b1c0be70ca2375788859ad39b64d61ed18db7392/drizzle/schema.ts#L54-L56